styles_about.css

```diff
@@ -205,10 +210,83 @@ body {
 .past-intro {
   margin-top: 0;
   text-align: center;
+  border: 0;
+  height: 10vw;
+  margin-left: -3vw;
+}
+
+#past-images {
+  width: 100vw;
+  height: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  margin-top: 2vw;
+}
```

Comment:
- [ ] Well done implementing the previous reviewer's suggestion 🥇 However, for mobile version, the images are still not behaving according to the given design guidelines 😓 See the images below for more clarity 👇 One possible way to fix this is adding height to `past-image2` only for the mobile version. Please note that this is just an example. You are not bound to use this 😅

Mobile version ❌ | Desktop version ✔
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235284652-a46cdb48-d2c1-4e24-b3a6-0cd03d43cec6.png) | ![image](https://user-images.githubusercontent.com/106119105/235284669-a56f9b27-3da9-4319-a239-7791ddbfd9cb.png)